resources/smcp-templates/v2.0/base

```diff
@@ -24,7 +24,7 @@ spec:
     level: warn
 
   proxy:
-    autoInject: true
+    autoInject: disabled
```

Comment:
why `disabled`? I think this has to be `false`, otherwise it will cause the `if` condition to be `true`

Reply:
doh! mixing contexts: new smcp and old values.yaml.  that saved an extra build/test cycle.  thanks!